packages/flake-tracker/src/jest/tracker.js

```diff
@@ -26,9 +26,12 @@ class JestFlakeTracker {
       // to call `makeRunResult`. This converts our `TestEntry` objects into `TestResults`
       // which have formatted errors
       const describeBlock = buildFlakeyDescribe(clone(state.rootDescribeBlock), this.flakes)
-      makeRunResult(describeBlock, state.unhandledErrors)
-        .testResults.filter((tr) => tr.status === 'flakey')
-        .forEach((tr) => db.writeErrors(getTestIDFromTestPath(tr.testPath), tr.errors))
+      try {
+        makeRunResult(describeBlock, state.unhandledErrors)
+          .testResults.filter((tr) => tr.status === 'flakey')
+          .forEach((tr) => db.writeErrors(getTestIDFromTestPath(tr.testPath), tr.errors))
+      } catch {} // ignoring weird behavior: "TypeError: Cannot read property 'replace' of undefined" that sometimes throws here
```

Comment:
by catching around the entire loop it will cause it all to be skipped. 

alternatively it could be try/catched in each loop

Reply:
the error is actually being thrown for makeRunResult